src/api/tests/test_available.py

```diff
@@ -101,15 +101,16 @@ def test_available_domain_dotgov(self):
         self.assertTrue(json.loads(response.content)["available"])
 
     def test_error_handling(self):
-        """Calling with bad strings raises an error."""
+        """Calling with bad strings returns error (error verifying) or unavailable (invalid domain)."""
```

Comment:
I think the error message portion should be a different test since it's a output of the api we are checking